#!/usr/bin/env python
PACKAGE = "gazebo_usar_core"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("point_cloud_publish_rate", double_t, 0, "Rate of point cloud publishing", 20.0, 1, 60)
gen.add("radiation_check_rate", double_t, 0, "Rate of radiation areas checking", 2.0, 1, 60)
gen.add("chemical_check_rate", double_t, 0, "Rate of chemical areas checking", 2.0, 1, 60)

exit(gen.generate(PACKAGE, "gazebo_usar_core", "Contamination"))
